java/test/src/main/java/io/ray/test/RuntimeEnvTest.java

```diff
@@ -40,7 +39,6 @@ public void testPerJobEnvVars() {
 
   public void testPerActorEnvVars() {
```

Comment:
I think you can just delete the test case.

Reply:
done